src/gltf-component-mappings.js

```diff
@@ -378,3 +378,11 @@ AFRAME.GLTFModelPlus.registerComponent("trimesh", "trimesh", el => {
 AFRAME.GLTFModelPlus.registerComponent("particle-emitter", "particle-emitter");
 
 AFRAME.GLTFModelPlus.registerComponent("networked-drawing-buffer", "networked-drawing-buffer");
+
+AFRAME.GLTFModelPlus.registerComponent(
+  "scene-audio-settings",
```

Comment:
perhaps just `audio-settings`? could imagine this being used at lower level nodes at some point (eg the regions we talked about)

Reply:
I wouldn't use the same component for regions. They would have different functionality so I'd use another name, but yeah we can remove scene from the name. @netpro2k what do you think?